### PR TITLE
[site] Resources are listed shifted right and note aligned correctly #7

### PR DESCRIPTION
**Pull request by jvega-crafter**
https://github.com/craftersoftware/craftersoftware-blueprint/issues/7

### CODE REVIEW

File: static-assets/css/custom.css [hunk 1]
```diff
@@ -63,9 +63,17 @@
      Resources
 --------------- */
 
+#portfolio {
```
Comment (rart) [inline static-assets/css/custom.css]: Don't target the original DOM paths without further differential scoping of your new style entries. For this entry, create your own classes and apply to the elements as necessary (i.e. rename this rule to `.porfolio-ext` and apply to the `#portfolio` element).

If portfolio-item (below) is a original class, please also replace for a new custom class.


FYI. This whole file needs to be broken apart into pieces individually imported in style.scss (just like "custom.scss" is). We need a ticket for this.

**Comment by rart** [comment]
Use these as reference and, as mentioned, try to revert things closer to the original
http://themes.semicolonweb.com/html/canvas/
http://themes.semicolonweb.com/html/canvas/portfolio-masonry.php

**Comment by jvega-crafter** [comment]
Thanks, templates are really helpfull, will do that

File: static-assets/js/functions.js [hunk 2]
```diff
@@ -1899,6 +1899,8 @@ var SEMICOLON = SEMICOLON || {};
 					}
 				}, 300);
 			});
+
+			$( "#portfolio").trigger( "grid-init" );
```
Comment (rart) [inline static-assets/js/functions.js]: Please add space after closing double quote of portfolio so that the code formatting is consistent.

Comment (rart) [inline static-assets/js/functions.js]: Note: Using this ID ties down the implementation to require that ID and element to be present. A better solution would be to trigger this event over `$( document )` and pass the current initialization ID for listeners. Checking the rest of the original source, the author hard coded other IDs so no point refactoring this now. Just FYI.

File: templates/web/resources.ftl [hunk 3]
```diff
@@ -68,8 +68,10 @@
                     ============================================= -->
                     <ul id="portfolio-filter" class="portfolio-filter clearfix" data-container="#portfolio">
                         
-                        <li :class="{activeFilter: !selectedType}"><a @click="changeType(null)">All</a></li>
-                        <li :class="{activeFilter: selectedType==key}" v-if="labels.type[key] && value > 0" v-for="(value, key) in types"><a @click="changeType(key)">{{ labels.type[key] }}</a></li>
+                        <li :class="{activeFilter: !selectedType}"><a @click="changeType()" data-filter="*">All</a></li>
+                        <li :class="{activeFilter: selectedType==key}" v-if="labels.type[key] && value > 0" v-for="(value, key) in types">
+                            <a  @click="changeType()" :data-filter="'.' + key">{{ labels.type[key] }}</a>
+                        </li>
```
Comment (rart) [inline templates/web/resources.ftl]: Good job. More readable :)

File: templates/web/resources.ftl [hunk 4]
```diff
@@ -149,56 +152,71 @@
...
-                         self.pagination.hasPrev = self.pagination.current != 1;
-                         self.pagination.hasNext = self.pagination.current + 1 <= self.pagination.total;
-                         self.$nextTick(function(){
-                         	if(window.studioICERepaint != undefined) {
-                            	window.studioICERepaint();
-                            }
-                         });
+
+                    Rx.Observable.forkJoin(cat)
+                    .subscribe(function (responses) {
```
Comment (rart) [inline templates/web/resources.ftl]: Please indent this whole block as
```
Rx.Observable.forkJoin(cat)
    .subscribe(function (response) {
        ...
    })
```

File: templates/web/resources.ftl [hunk 5]
```diff
@@ -149,56 +152,71 @@
...
+
+                        var resources = responses[0].body;
+                        self.types = resources.facets.type_s;
+                        self.tags = resources.facets["tags.item.key"];
+                        self.pagination.total = Math.ceil(parseInt(resources.total) / self.pagination.rows);
+                        self.pagination.hasPrev = self.pagination.current != 1;
+                        self.pagination.hasNext = self.pagination.current + 1 <= self.pagination.total;
+                        responses.shift();
+
+                        var resources = { items : [] };
```
Comment (rart) [inline templates/web/resources.ftl]: Remove `var`

File: templates/web/resources.ftl [hunk 6]
```diff
@@ -149,56 +152,71 @@
...
+
+                        for(var x = 0; x < responses.length; x++){
+                            var items = responses[x].body.items;
```
Comment (rart) [inline templates/web/resources.ftl]: Fix indentation/code format

File: templates/web/resources.ftl [hunk 7]
```diff
@@ -149,56 +152,71 @@
...
+                                for(var y = 0; y < items.length; y++){
+                                    items[y].type = self.typesArray[x];
+                                }
+                                $.merge( resources.items , responses[x].body.items )
+                        }
+                        self.resources = resources;
+
+                        if(self.selectedType){
+                            $( "#portfolio" ).on( "grid-init", function() {
+                                $('#portfolio').isotope({ filter: '.' + self.selectedType });
```
Comment (rart) [inline templates/web/resources.ftl]: Stick to only one format; either space between quotes (e.g. line 202) or no spaces (e.g. line 203)

File: templates/web/resources.ftl [hunk 8]
```diff
@@ -149,56 +152,71 @@
...
-                changeType: function(newType) {
-                	this.selectedType = newType;
-                    this.load();
+                changeType: function() {
+                    if(window.studioICERepaint != undefined) {
+                        window.studioICERepaint();
+                    }
                 },
                 changeTag: function(newTag) {
                 	this.selectedTag = newTag;
```
Comment (rart) [inline templates/web/resources.ftl]: Indentation

**Pull request merged**